src/runway/path.py

```diff
@@ -0,0 +1,119 @@
+""" Runway configuration 'path' settings """
+import os
+import logging
+import six
+
+from .sources.git import Git
+
+LOGGER = logging.getLogger('runway')
+
+
+class Path(object):
+    """ Runway configuration 'path' settings object """
+
+    def __init__(self, module, env_root, cache_dir=None):
+        """ Initialize. """
+        if not cache_dir:
+            cache_dir = os.path.expanduser("~/.runway_cache")
+
+        self.env_root = env_root
+        self.cache_dir = cache_dir
+        self.source, self.uri, self.location, self.options = self.parse(module)
+        self.module_root = self.__get_module_root_dir(module)
+
+    def parse(self, module):
+        """ Retrieve the source and location of the path variable. """
+        source = 'local'
+        uri = ''
+        location = ''
+        options = ''
+
+        split_source_location = module.get('path').split('::')
```

Comment:
As is, it will thrown an error if `module['path']` does not exist because `.get()` will return `None` which does not have the method `split()` so either do this

```suggestion
        split_source_location = module.get('path', '').split('::')
```

or this with handling for `KeyError` if path does not exist. Which should result in `sys.exit(1)` after an error message is logged.

```suggestion
        split_source_location = module.get['path'].split('::')
```

Reply:
What would be the preferred approach? The first example is very similar to what I've done with past languages, so I'd lean towards that.

Reply:
It looks like what you have _should_ be able to handle the fist option as is so go for it.